pkg/controller/controller.go

```diff
@@ -166,6 +168,22 @@ func (c *Controller) Run(stopCh <-chan struct{}) {
 	// add all admin domain and system namespaces to the queue initially
 	c.cron.AddAdminSystemDomains()
 
+	// create a configmap for the latest timestamp to contact zms
+	_, err := c.clientset.CoreV1().ConfigMaps("kube-yahoo").Get("athenzcall-config", metav1.GetOptions{})
+	if apiErrors.IsNotFound(err) {
```

Comment:
Do we need an nil error check here? ex: `err != nil api && Errors.IsNotFound(err)`?